### PR TITLE
nwc_health_report attribute requires a value

### PR DESCRIPTION
**Pull request by Tontonitch**
ITL - Fix for nwc_health_report which should be a value

### CODE REVIEW

File: itl/plugins-contrib.d/network-components.conf [hunk 1]
```diff
@@ -604,7 +604,7 @@ object CheckCommand "nwc_health" {
 			description = "The role of this device in a hsrp group (active/standby/listen)"
 		}
 		"--report" = {
-			set_if = "$nwc_health_report$"
+			value = "$nwc_health_report$"
```
Comment (dnsmichi) [inline itl/plugins-contrib.d/network-components.conf]: Which value can be set? The docs are unclear about that.

**Comment by dnsmichi** [comment]
Hi, thanks for the patch. I don't know how that happened, but you've got a lot of dangling commits in your PR. Can you please clean that up (and also fix the value docs), by branching off the current master, and then re-applying your patch for a PR? Thanks a lot.

File: itl/plugins-contrib.d/network-components.conf (hunk 1 above)
Comment (Tontonitch) [inline itl/plugins-contrib.d/network-components.conf]: values can be "long" (default), "short" (to shorten if available), or "html" (to produce some html outputs if available)

**Comment by Tontonitch** [comment]
> I don't know how that happened, but you've got a lot of dangling commits in your PR. Can you please clean that up (and also fix the value docs), by branching off the current master, and then re-applying your patch for a PR?

I will see. I followed the procedure https://help.github.com/articles/syncing-a-fork/ to update my fork before doing the change. I don't know why I had all these commits when I did the pull request.

File: itl/plugins-contrib.d/network-components.conf (hunk 1 above)
Comment (dnsmichi) [inline itl/plugins-contrib.d/network-components.conf]: Would be cool if you can add that to both, the description field and also the docs :)

**Comment by dnsmichi** [comment]
I guess that history diverged somehow from your local master compared to our remote origin and now you've got a merged tree. I guess that happened because of a previous patch I did cherry-pick/amend and directly merge your PR - commit ids different, and as such, different history.

**Comment by Tontonitch** [comment]
Sorry for the delay. It should be okay now.

**Comment by dnsmichi** [comment]
Cool, thanks 👍

**Comment by dnsmichi** [comment]
Thanks, I've picked your commits and squashed them into one commit. That one reaches the git master.

One thing I'd suggest to (and will add that to contribution guides soon):

- do not add things in master, but leave that as upstream source
- branch off an uptodate master, and add your commits in a dedicated fix or feature branch
- create your pull request based on that branch

That way you'll only update the master branch from time to time, but never have to rebase your fix/feature branches, unless someone asks you to do so.

**Pull request closed without merging**